l10n/en/firefox/all.ftl

```diff
@@ -39,7 +39,11 @@ firefox-all-test-your-sites-against = Test your sites against soon-to-be-release
 firefox-all-count-on-stability-and = Count on stability and ease of use with this { -brand-name-firefox } browser built for enterprise.
 firefox-all-windows-installers-for = Windows installers for corporate IT that simplify the configuration, deployment and management of the { -brand-name-firefox-browser }.
 firefox-all-arm64-installers = ARM64/AArch64 installers
+firefox-all-arm64-installers-optimized-v2 = ARM64/AArch64 installers optimized for Linux and Snapdragon-powered Windows PCs.
+
+# Obsolete string (expires: 2024-06-04)
 firefox-all-arm64-installers-optimized = ARM64/AArch64 installers optimized for Snapdragon-powered { -brand-name-windows } PCs.
```

Comment:
Before this gets to pontoon: When we're touching the string now, is there a chance this can be simplified to be more future-proof, dropping the Qualcomm brand in case there are more / it changes in the future, and referring more to the ARM architecture, maybe talking about it as "SoC-powered" or something like this? (I'm not an expert in this matter so maybe @gabrielBusta could jump in with some lingo tips, not only for the Linux part but also making the Windows bit be more generic for future use?…)

Reply:
Perhaps we should just keep it simple and say `ARM64/AArch64 installers optimized for Linux and Windows PCs.`